help/mobile-notifications.md

```diff
@@ -43,6 +47,29 @@ with Zulip notifications on your Android phone, we recommend Signal's excellent
 [troubleshooting guide](https://support.signal.org/hc/en-us/articles/360007318711-Troubleshooting-Notifications#android_notifications_troubleshooting),
 which explains the notification settings for many popular Android vendors.
 
+### Contacting Zulip support
+
+If you are still having trouble with your push notifications, you can send an
+email to [Zulip support](/help/contact-support). Please be sure to include the
+troubleshooting data provided by the mobile app.
+
+{start_tabs}
+
+{tab|mobile}
+
+{!mobile-profile-menu.md!}
+
+1. Tap **Settings**.
+
+1. Tap **Notifications**.
+
+1. Tap **Troubleshooting**.
+
+1. Use the **Copy to clipboard** button or the **Email support@zulip.com**
+   button to email us the troubleshooting data.
```

Comment:
Let's say "... to email the troubleshooting data to the Zulip support team." to avoid using the second person here.